src/cmd/input.go

```diff
@@ -28,6 +29,16 @@ func readInputConfig() {
 	viper.ReadInConfig()
 }
 
+func readResourceInput[T any]() (*T, error) {
+	readInputConfig()
+	var evt *T
```

Comment:
This should start out as a `T` and then be returned at the return as `*evt`

Reply:
Good call 👍